test/e2e/e2e_test.go

```diff
@@ -300,6 +300,32 @@ var _ = Describe("controller", Ordered, func() {
 			}
 			EventuallyWithOffset(1, verifyPodResources, 3*time.Minute, 15*time.Second).Should(Succeed())
 		})
+		It("change minReadySeconds", func() {
+			cmd := exec.Command("kubectl",
+				"-n", namespace,
+				"patch", "valkeycluster", "valkeycluster-sample",
+				"--type=json",
+				`-p=[{"op":"replace","path":"/spec/minReadySeconds","value":15}]`,
+			)
+			_, err := utils.Run(cmd)
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+			verifyClusterMinReadySeconds := func() error {
```

Comment:
This will verify that the ValkeyCluster object has a new value for minReadySeconds, but it will not verify that the statefulsets have this value applied.

Reply:
Ah true, I'll make the testcase stronger. I did wonder too (as with other testcases) that there isn't any guarantee that the change has applied if the default value happens to match the test value. In that unlikely case, a no-op and a success would both pass I suppose.